### ISSUE TITLE
[DataCap Application] Eorange

### ISSUE THREAD

**Issue by Gina520** [filecoin-project/filecoin-plus-large-datasets#566]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: Eorange Network
- Website / Social Media: www.eorange.com.cn
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB):100TiB
- On-chain address for first allocation:f1drs7od3s3dl24kobll2rxqxyk2ngfcvssdyx5fa

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Eorange was established on August 18, 2011 with a registered capital of 110 million RMB, mainly engaged in communication infrastructure construction and maintenance, Internet value-added business, comprehensive outsourcing services in communication industry, ICT, software R&D (VR, AR).
```

What is the primary source of funding for this project?
```
Own financial support.
```

What other projects/ecosystem stakeholders is this project associated with?
```
No
```

## Use-case details

Describe the data being stored onto Filecoin
```
It were aerial surveillance videos.
```

Where was the data in this dataset sourced from?
```
From our UAV products' recording.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
[1.zip](https://github.com/filecoin-project/filecoin-plus-large-datasets/files/9250715/1.zip)
[11.zip](https://github.com/filecoin-project/filecoin-plus-large-datasets/files/9250716/11.zip)
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Confirm.
```

What is the expected retrieval frequency for this data?
```
We plan retrieve data twice a year.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
On Filecoin, we'll keep storing 2 years.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
In China.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Both offline and online transfers. More offline transfer. 
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We'll focus on sp's work time and whether they support retrieval.
```

How will you be distributing deals across storage providers?
```
Different distribution according to distance.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes.
```

**Comment by Sunnyiscoming** [comment]
@Gina520 Hey. Could you send an email to filplus-app-review@fil.org with your official domain in order to confirm your identity?

**Comment by Gina520** [comment]
@Sunnyiscoming ok ![34a2c96](https://user-images.githubusercontent.com/107161183/186489263-e20fdad3-aa62-4138-a295-aa11e880229a.jpg)

**Comment by Gina520** [comment]
@Sunnyiscoming ok.
![34a2c96](https://user-images.githubusercontent.com/107161183/186489960-1014109e-d88c-42c8-92e4-7483a8891604.jpg)

**Comment by Sunnyiscoming** [comment]
Received that.

**Comment by Gina520** [comment]
@raghavrmadya Hi, I don't know who he is. Can you please tell me what you think?

**Comment by raghavrmadya** [comment]
<img width="847" alt="Screen Shot 2022-08-30 at 1 33 20 PM" src="https://user-images.githubusercontent.com/91579519/187537745-3e98867c-662d-4be6-b0b0-7cfcda148f8a.png">

**Comment by raghavrmadya** [comment]
Also we need further clarification on how this dataset is public as it contains surveillance footage

**Comment by Gina520** [comment]
@raghavrmadya With your pointing out, now I notice the mistake I made in my issue. I'm new to the community. Before I posted my issue, I had studied in the community to learn how other members write issues. However, I accidentally copied what I saw in the community when I was writing it. I didn't notice this problem when I first published the issue, so thank you for the careful correction!

All videos data are chosen from our company's UAV products. The reason we made and stored these videos is because we worked with electric power company. We help them patrol the power patrol area, get a comprehensive and accurate image of overhead cables and transmit the cable images to staff. So we need to store these videos for the power company's needs.

**Comment by raghavrmadya** [comment]
So you are not the client but you are storing on someone's behalf?

**Comment by raghavrmadya** [comment]
Your application previously mentioned Hiscene but you have since made an edit. Please explain

**Comment by Gina520** [comment]
@raghavrmadya Did you receive my email to filplus-app-review@fil.org before? I'm a staff of Eorange. The data sample I provided is videos about recording of our engineering area. Hope you can double check.
The reason I mentioned Hiscene before is that I'm not very familiar with how to fill out this issue, I quoted some of their descriptions. This is my mistake, I have checked and corrected it.
We need to store our data via Filecoin and hope we can apply and receive datacap successfully. We do want to participate in Filecoin. Thanks again.

> @raghavrmadya With your pointing out, now I notice the mistake I made in my issue. I'm learning something in the community. Before I posted my issue, I had studied in the community to learn how other members write issues. However, I accidentally copied what I saw in the community when I was writing it. I didn't notice this problem when I first published the issue, so thank you for the careful correction!
> 
> All videos data are chosen from our company's UAV products. The reason we made and stored these videos is because we worked with electric power company. We help them patrol the power patrol area, get a comprehensive and accurate image of overhead cables and transmit the cable images to staff. So we need to store these videos for the power company's needs.

**Comment by Sunnyiscoming** [comment]
Can you provide more data samples related with the data you will store?  How many copies will you store?
What's the relationship between you and organization?
Can you provide more detailed information about other storage providers participated in this program, such as you can list SPs you have contacted with at present?

**Comment by raghavrmadya** [comment]
Waiting on client response.

**Comment by EorangeOfficial** [comment]
@Sunnyiscoming @raghavrmadya Hello! We just finished our vacation, so I come to give our response this time!
@Gina520 is a staff of our company and she once mentioned it. https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/566#issuecomment-1240754836 We have made a change in position and Gina is no longer in charge of this application. I will be in charge of it next.
Here's the data samples. https://drive.google.com/file/d/1TWdmQhuBHEV5Bl0HLrakI9CoEO0rkjOE/view?usp=sharing We plan store 8 copies since we have 625TiB data total.
We've contacted with f062260, f0752695, f01199442 at present. (China, Korea, Europe) More SP will be contacted in the next step.
